src/pages/SonarFeed/SonarFeedActivityPage.js

```diff
@@ -79,57 +70,31 @@ const SonarFeedActivityPage = ({ activities, isLoading, classes = {} }) => {
 
   return activities && activities.length ? (
     <div className={cx(styles.wrapper, classes.activitiesWrapper)}>
-      {activities.map(
-        (
-          { triggeredAt, payload, trigger: { id: signalId, title } = {} },
-          index
-        ) => (
-          <div
-            key={triggeredAt + '_' + signalId}
-            className={cx(
-              styles.activityItem,
-              classes.activityItem,
-              index === 0 && classes.firstActivity
-            )}
-          >
-            <div className={cx(styles.description, classes.activityCustom)}>
-              <h4 className={styles.date}>
-                {formatDate(triggeredAt)} by{' '}
-                <Link
-                  to={`/sonar/signal/${signalId}${SIGNAL_ANCHORS.ACTIVITIES}`}
-                  className={styles.link}
-                >
-                  {title}
-                </Link>
-              </h4>
-            </div>
-            <Markdown
-              source={Object.values(payload)[0]}
-              className={classes.activityMarkdown}
-            />
-          </div>
-        )
-      )}
+      {activities.map((item, index) => (
+        <ActivityRenderer activity={item} index={index} classes={classes} />
+      ))}
     </div>
   ) : (
     <SonarFeedRecommendations description='There are not any activities yet' />
   )
 }
 
-const enhance = graphql(TRIGGER_ACTIVITIES_QUERY, {
-  options: () => ({
-    variables: {
-      datetime: new Date().toISOString()
-    },
-    fetchPolicy: 'cache-and-network'
-  }),
-  props: ({ data }) => {
-    return {
-      activities: (data.activities || {}).activity,
-      isLoading: data.loading,
-      isError: !!data.error
+export const getActivitiesEnhance = () => {
+  return graphql(TRIGGER_ACTIVITIES_QUERY, {
```

Comment:
you can skip `return` if you use return only in body

Reply:
@haritonasty , i know. But it is very usefull for 'debugger'. Not critical